design/README.md

```diff
@@ -44,37 +40,44 @@ The expanded version should be use to generate the SVG file.
--   To push your changes, first ensure you're on your feature branch _*and not*_ `master` and then push to `origin/my-branch-name`
-    -   In SourceTree press the Push button in the toolbar and follow the prompts.
-    -   In the command line, use `git push origin\my-branch-name`. The `-u` flag can be used to set upstream for that branch.
--   Create a Pull Request to begun the process of merging your branch into master.
-    -   Open [the repo in Bitbucket](http://esgit.tcc.etn.com:7990/projects/BLUE/repos/icons/browse) and click the "Create pull request" button in the left side toolbar.
-    -   Select the source branch, e.g. `my-branch-name` and ensure the destination is `master`.
-    -   In the Description box, enter any relevant information about the pull request/feature/icon. For instance, if your icon "breaks" Material Design guidelines for good reason, please explain why.
-    -   Add Reviewers to the pull request. Include `Atha, Eric` and `Boyle, Joseph P`. This may change as the design and PXBlue Team grows.
--   The reviewers will either approve, decline or mark as needs work.
-    -   If approved, the reviewers will merge into `master`
-    -   If declined, the reviewers will comment why
-    -   If marked as needs work, the reviewers will comment why and will give the designer time to revise and update the pull request.
+The following instructions are for individuals who are comfortable working with GitHub and the command line on their computer. If you prefer, you can send new icons directly to the PX Blue team via email and we will take care of this part of the process for you.
+
+In order to work with this GitHub repository, you will need to have a github account (if you don't have one, you can sign up for free).
+
+For outside contributors, please refer to our [Contribution Guidelines](https://github.com/pxblue/.github/blob/master/CONTRIBUTING.md#get-started). To summarize the steps for adding your icons and creating a Pull Request for approval:
+- Fork the icons repository to your account
+    - Go to https://github.com/pxblue/icons and click the Fork button in the top right of the screen (outside contributors who have been added to the icons repository can clone directly from the PX Blue repository without forking)
+- Clone the repository to your local computer
+    ```
+    git clone https://github.com/<your-user-id>/icons
+    cd icons
+    git remote add upstream https://github.com/pxblue/icons
+    git pull upstream dev
+    ```
+-   Create a new Branch for your changes. The name of the branch should start with `feature/` followed by the name of your icon (or a simple summary if you are including multiple icons), e.g., `feature/ups`. If your icon name is multiple words, you must separate them with a dash.
+    ```
+    git checkout -b feature/icon
+    ```
+- Make your changes in the design folder of your local copy of the repository
```

Comment:
Is modifying META.json also part of their duty?

Reply:
Or is it on our side when we do yarn build?

Reply:
If they are emailing it to us, it would be on us...if they are contributing directly, it would be on them. I'll update the content